### DISCUSSION TITLE
payment error

### DISCUSSION THREAD

**Discussion by kolenval2023** [coinfoundry/miningcore#1693] [Q&A] [answered]
Hi all! Set up a pool and it successfully works. But problems arose later. 
1) In the web interfaces on the "payments" tab, the found blocks are not displayed
2) An error appeared in the miner log.
[E] [bitcoin-cash] [Bitcoin Payout Handler] sendmany returned error: Wallet has insufficient funds code -6 
Can you please tell me what should be done to correct these errors? I will provide additional information if necessary.
p.s.
Block rewards go to the correct address, no problem with that. In the BCH node application, they are displayed correctly.

**Comment by ghost** [comment, marked as answer]
Try running the daemon with the "-fallbackfee=0.00001" flag.

**Reply by kolenval2023** [reply]
how to contact you?
